### ISSUE TITLE
"⌘ P to search" needs to be accessible.

### ISSUE THREAD

**Issue by yzen** [devtools-html/debugger.html#4069]
At the moment "⌘ P to search" area is not accessible to screen readers. It is however clickable to users that use the mouse. This interaction should also be possible using just the keyboard and for that:
* "⌘ P to search" should be focus-able with the keyboard
* Should have styling when it is hovered and focused
* Should be activate-able with the keyboard by pressing Enter/Space when it is in focus.
* Should have appropriate semantics. It looks like "⌘ P to search" is a button that triggers a focus jump to search field so it should either be a <BUTTON> or should have an ARIA role of button.

**Comment by jasonLaster** [comment]
we're removing the link.
